TodoList/AppDelegate.swift

```diff
@@ -15,6 +15,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         return true
     }
-
+    
+    func applicationDidEnterBackground(_ application: UIApplication) {
+        UserDefaults.standard.synchronize()
```

Comment:
https://developer.apple.com/documentation/foundation/userdefaults/1414005-synchronize
> Waits for any pending asynchronous updates to the defaults database and returns; **this method is unnecessary and shouldn't be used.**

And more:
> -synchronize is deprecated and will be marked with the NS_DEPRECATED macro in a future release.
> -synchronize blocks the calling thread until all in-progress set operations have completed. This is no longer necessary. Replacements for previous uses of -synchronize depend on what the intent of calling synchronize was. If you synchronized...
> - ...before reading in order to fetch updated values: remove the synchronize call
> - ...after writing in order to notify another program to read: the other program can use KVO to observe the default without needing to notify
> - ...before exiting in a non-app (command line tool, agent, or daemon) process: call CFPreferencesAppSynchronize(kCFPreferencesCurrentApplication)
> - ...for any other reason: remove the synchronize call
> open func synchronize() -> Bool

If it wasn't clear enough, here's an Apple Engineer:
https://twitter.com/Catfish_Man/status/1041754743645732864
>Foundation release notes for iOS 12 are up at https://developer.apple.com/documentation/ios_release_notes/ios_12_release_notes/foundation_release_notes
> Of particular note, and this one’s for @ctp: -[NSUserDefaults synchronize] is truly dead now. No longer does anything except return a bool.
>*pours one out for our grand decade+ war against that stupid bool*

And for the joke:
https://twitter.com/realbadiostips/status/1042143807083405312
> In iOS 12: [NSUserDefaults synchronize] actually increases UserDefaults performance. I recommend putting it in applicationDidBecomeActive.